sender/sms/sms.go

```diff
@@ -1,11 +1,27 @@
 package sms
 
 import (
+	"errors"
+	"fmt"
 	gotell "github.com/ftpsolutions/go-tell"
 	"github.com/ftpsolutions/go-tell/sender"
+	"regexp"
+	"strings"
 )
 
 func validateSMS(job *gotell.Job) error {
+	re, _ := regexp.Compile(`^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$`)
```

Comment:
I think this pattern will only solve for Australia numbers- that's probably fine in the short term, but maybe somebody has made a nice pattern (or list of patterns) that matches mobile numbers globally?